docs/my-website/docs/observability/langtrace_integration.md

```diff
@@ -49,15 +41,13 @@ print(response)
 
 ```yaml
 model_list:
-  - model_name: gpt-4
+  - model_name: "gpt-4"
     litellm_params:
-      model: openai/fake
-      api_key: fake-key
-      api_base: https://exampleopenaiendpoint-production.up.railway.app/
+      model: openai/gpt-4
 
 litellm_settings:
-  callbacks: ["langtrace"]
```

Comment:
we recommend using callbacks here - did this not work for you ?

Reply:
it worked with just `callbacks:` will adjust now